### PR TITLE
Polish translations

### PR DESCRIPTION
**Pull request by gienkov**

### CODE REVIEW

File: addons/frag/stringtable.xml [hunk 1]
```diff
@@ -3,33 +3,43 @@
...
             <English>Maximum Projectiles Tracked</English>
+            <Polish>Maks. liczba śledzonych pocisków</Polish>
         </Key>
         <Key ID="STR_ACE_frag_MaxTrack_Desc">
-            <English>This setting controls the maximum amount of projectiles the fragmentation and spalling system will track at any given time. If more projectiles are fired, they will not be tracked. Lower this setting if you do not want FPS drops at high-count projectile scenarios ( >200 rounds in the air at once)</English>
+            <English>This setting controls the maximum amount of projectiles the fragmentation and spalling system will track at any given time. If more projectiles are fired, they will not be tracked. Lower this setting if you do not want FPS drops at high-count projectile scenarios ( &gt;200 rounds in the air at once)</English>
+            <Polish>To ustawienie kontroluje maksymalną ilość pocisków, jakie fragmentacja i odpryski symulują w danym momencie. Jeżeli więcej pocisków będzie wystrzelonych, wtedy nie będą one śledzone. Zmniejsz tą opcję jeżeli nie chcesz odczuwać spadków FPS podczas ciężkiej wymiany ognia (więcej niż 200 pocisków w powietrzu na raz).</Polish>
```
Comment (gienkov) [inline addons/frag/stringtable.xml]: Tabler did that, not me

Comment (bux578) [inline addons/frag/stringtable.xml]: That's fine. `>` and `<` are illegal characters in xml.

**Comment by gienkov** [comment]
Can this go in already?

**Comment by gienkov** [comment]
Translated new things too

**Pull request merged**